TW 435 Audit logs page (#3996)
postmanlabs/postman-docs: src/pages/docs/administration/audit-logs.md

> Audit logs are visible to all [team admins](/docs/collaborating-in-postman/roles-and-permissions/#team-roles) on [Postman Professional and Enterprise plans](https://www.postman.com/pricing), including super admins on Enterprise teams.

## Contents

* [Viewing audit logs in the dashboard](#viewing-audit-logs-in-the-dashboard)

* [Exporting audit logs from the dashboard](#exporting-audit-logs-from-the-dashboard)

* [Accessing audit logs via the Postman API](#accessing-audit-logs-via-the-postman-api)

* [Logged events](#logged-events)

## Viewing audit logs in the dashboard

You can view audit logs in Postman by selecting **Team** in the header, then selecting **Audit Logs**.

<img alt="Team menu with audit logs selected" src="https://assets.postman.com/postman-docs/menu-select-audit-logs-v9.13.0.jpg" width="300px"/>

Audit logs provide information about actions taken by members of your team. Each event includes the user, the user's email address, the event name and a description of the event, the user's IP address, and the date and time of the action, including the user's timezone.

> Use the filters to filter the actions by date range, event type, and user.

Reviewer: Maybe something like this to avoid using filter twice
```suggestion
You can use filters to view actions by date range, event type, and user.
```

Author: Good call! This is much better 😁

| Sent Team Invite  | An invitation was sent to a user to join your team.  |
| Approved Team Invite | An admin approved an email invitation. |
| Rejected Team Invite  | An admin rejected an email invitation. |

> | Canceled Team Invite   | An invitation for a user was canceled.   |

Reviewer 2: For all three of these - I am not sure we can switch from "cancelled" to "canceled" because it's spelled wrong in the UI.

Author: Oh, right! Duh 😁 🤦  I'll revert.
